From 2690b8deb0578399787035a142aea9fa7b71de53 Mon Sep 17 00:00:00 2001
From: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
Date: Mon, 29 Jul 2013 11:53:31 +0900
Subject: board: koelsch: Fix u-boot start address

U-boot start address is changed from 0x40080000 to 0xE6304000.
And U-boot initial stack is changed from 0x4027fffc to 0xE633fffc.

Signed-off-by: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 include/configs/koelsch.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/koelsch.h b/include/configs/koelsch.h
index f9f1131..ff4571c 100644
--- a/include/configs/koelsch.h
+++ b/include/configs/koelsch.h
@@ -75,7 +75,7 @@
 #define CONFIG_TMU_TIMER
 
 /* STACK */
-#define CONFIG_SYS_INIT_SP_ADDR		0x4027fffc
+#define CONFIG_SYS_INIT_SP_ADDR		0xE633fffc
 #define STACK_AREA_SIZE				0xC000
 #define LOW_LEVEL_MERAM_STACK	\
 		(CONFIG_SYS_INIT_SP_ADDR + STACK_AREA_SIZE - 4)
@@ -140,7 +140,7 @@
 #define CONFIG_SYS_FLASH_UNLOCK_TOUT	3000
 
 #else
-#define CONFIG_SYS_TEXT_BASE	0x40080000
+#define CONFIG_SYS_TEXT_BASE	0xE6304000
 #define CONFIG_SPI
 #define CONFIG_SH_QSPI
 #define CONFIG_SPI_FLASH
-- 
1.7.10.4

